e2e_samples/parking_sensors/README.md

```diff
@@ -25,9 +25,12 @@ The sample demonstrate how DevOps principles can be applied end to end Data Pipe
     - [Databricks](#databricks)
     - [Data Factory](#data-factory)
 - [How to use the sample](#how-to-use-the-sample)
-  - [Prerequisites](#prerequisites)
-    - [Software pre-requisites if you use dev container](#software-pre-requisites-if-you-use-dev-container)
-  - [Setup and Deployment](#setup-and-deployment)
+  - [Setup](#setup)
+    - [Prerequisites](#prerequisites)
+    - [Deployment Environment](#deployment-environment)
```

Comment:
can we just call this bullet `Deployment Options` instead of Deployment environment? it seems neater (there is some confusion with Deployment and Configuration too IMO)